xarray/core/dataarray.py

```diff
@@ -1175,6 +1175,35 @@ def fillna(self, value):
         out = ops.fillna(self, value)
         return out
 
+    def interpolate(self, dim=None, method='linear', inplace=False):
```

Comment:
I like calling the `interpolate_na` rather than `interpolate` to make it clear what this method does. See here for the related pandas discussion: https://github.com/pandas-dev/pandas/issues/9340